Tests/AutoCollection/RequestDataHelper.tests.ts

```diff
@@ -33,15 +33,49 @@ describe("AutoCollection/RequestDataHelper", () => {
         it("should return an absolute url", () => {
 			var helper = new RequestDataHelper(<any>request);
 			var requestData = helper.getRequestData();
-			assert.equal(requestData.baseData.url, "http://bing.com/search%3Fq=test");
+			assert.equal(requestData.baseData.url, "http://bing.com/search?q=test");
         });
 		
 		it("should return an absolute url for encrypted traffic", () => {
 			request.connection.encrypted = true;
 			
 			var helper = new RequestDataHelper(<any>request);
 			var requestData = helper.getRequestData();
-			assert.equal(requestData.baseData.url, "https://bing.com/search%3Fq=test");
+			assert.equal(requestData.baseData.url, "https://bing.com/search?q=test");
+        });
```

Comment:
May be add tests for some border case scenarios like when the actual path name is empty, operation name might have to be "GET /" as opposed to "GET ".